G/GDAL/build_tarballs.jl

```diff
@@ -0,0 +1,69 @@
+using BinaryBuilder
+
+name = "GDAL"
+version = v"3.0.2"
+
+# Collection of sources required to build GDAL
+sources = [
+    "https://github.com/OSGeo/gdal/releases/download/v$version/gdal-$version.tar.gz" =>
+    "787cf150346e58bff0ccf8c131b333139273e35d2abd590ad7196a9ee08f0039",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gdal-*/
+
+# Show options in the log
+./configure --help
+./configure --prefix=$prefix --host=$target \
+    --with-geos=${bindir}/geos-config \
+    --with-proj=$prefix \
+    --with-libz=$prefix \
+    --with-sqlite3=$prefix \
+    --with-curl=${bindir}/curl-config \
+    --with-python=no \
+    --enable-shared \
+    --disable-static
+
+make -j${nproc}
+make install
+"""
+
+platforms = supported_platforms()
```

Comment:
Also this libraries uses C++, shouldn't you expand C++ string ABIs also here?  That would also probably requiring `CompilerSupportLibraries_jll` unnecessary.

Reply:
Yeah I didn't add it before since I wasn't sure any of [these](https://github.com/JuliaPackaging/Yggdrasil/pull/267#issuecomment-564659866) criteria were met. I expanded the string ABIs and removed `CompilerSupportLibraries_jll` in fed5e11283b13ffe62acb1b794d23e4dec2c5cf3.